### PR TITLE
finished prep links excercise(james aguirre)

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-01-09 at 8 02 58 PM
Screenshot 2023-01-09 at 8 03 03 PM

### CODE REVIEW

File: prep-html-links/contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="http://localhost:5500/prep-html-links/index.html">Click here to go to the home page</a>
```
Comment (Zhiwutian) [inline prep-html-links/contact-us/contact-us.html:8]: you don't want to use `localhost` here, you just want the relative path from this file to the file you want to link to

File: prep-html-links/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="http://localhost:5500/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```
Comment (Zhiwutian) [inline prep-html-links/index.html:8]: same as above, make sure to update the path.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates are needed. Make sure to reach out if you have questions

File: prep-html-links/contact-us/contact-us.html (hunk 1 above)
Comment (james-aguirre) [reply prep-html-links/contact-us/contact-us.html]: @Zhiwutian Hi i updated the links, is this correct?
Screenshot 2023-01-10 at 1 25 26 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**